src/app/shared/search/search-filters/search-filter/search-filter.component.html

```diff
@@ -1,14 +1,15 @@
 <div class="facet-filter d-block mb-3 p-3" *ngIf="active$ | async"
-     [id]="regionId" [attr.aria-labelledby]="toggleId" [ngClass]="{ 'focus': focusBox }" role="region">
```

Comment:
It looks like you've accidentally removed accessibility attributes on this `<div>`.  We should add back in the `aria-labelledby` and `role` attributes, unless there's a clear reason why you've removed them.

Reply:
As I mentioned last week, you need to undo the changes to this file.  In this area you accidentally removed the `aria-labelledby` and `role` attributes which are required for accessibility.